forms/form/src/main.rs

```diff
@@ -112,12 +102,14 @@ mod tests {
             resp.headers().get(CONTENT_TYPE).unwrap(),
             HeaderValue::from_static("text/plain")
         );
-        assert_eq!(resp.body().as_str(), "Your name is John");
+
+        let body = to_bytes(resp.into_body()).await.unwrap();
+        assert_eq!(body.as_str(), "Your name is John");
     }
 
     #[actix_rt::test]
```

Comment:
We can use `actix_web:::test` everywhere (and drop the direct dependency on `actix_rt`).

Reply:
Thank you! I fixed it.